kong/globalpatches.lua

```diff
@@ -414,6 +414,7 @@ return function(options)
     local seeded = {}
     local randomseed = math.randomseed
 
+    _G.math.native_randomseed = randomseed
```

Comment:
this shouldn't be exposed. If needed for testing, then only expose it during tests.

Reply:
done , only expose it for busted environment